### ISSUE TITLE
1.4.3 still says NOR

### ISSUE THREAD

**Issue by greut** [mosquito/cysystemd#32]
Thanks for the release, however it seems that the NOR ~> OR didn't make it.

```python
>>> cysystemd.__version__
'1.4.3'

>>> dir(cysystemd.reader.MatchOperation)
['AND', 'NOR', '__class__', '__doc__', '__members__', '__module__']
```

`file = "cysystemd-1.4.3-cp37-cp37m-manylinux2014_x86_64.whl", hash = "sha256:5c246ecd2a262e18aa49146a52221c44941a87998962348b6446fc908fbd8f7f"`

**Comment by mosquito** [comment]
My mistake. I will fix it as soon as possible.

**Comment by mosquito** [comment]
Should be fixed in already released `1.4.4`.

**Comment by greut** [comment]
@mosquito much better indeed. Thanks for the quick follow up!
